scalyr_agent/third_party/tcollector/collectors/0/dfstat.py

```diff
@@ -79,7 +82,7 @@ def main():
             # Scalyr edit.  We've found some systems list the same mount point multiple times in the df output.
             # To prevent duplicte timestamp warnings, we keep track of which mount points we've reported.
             reported_mounts = {}
-            for line in stdout.split("\n"): # pylint: disable=E1103
+            for line in stdout.decode("utf-8").split("\n"): # pylint: disable=E1103
```

Comment:
I'm pretty sure all of these should be just straight ascii.  If there is high-ascii, I definitely think it's not meant to be interpreted as UTF-8.  Knowing this, would it be better to decode as ascii?

Reply:
It looks like the UTF-8 is the default encoding for the majority of modern operating systems. Linux kernel also uses UTF-8 to work with text, so I suppose, standard command-line tools should be switched to UTF-8 too. 

And one more thing. If there is an extended ASCII encoded text, we still can not use "ascii" and should use "latin-1" right?

Reply:
@ArthurKamalov I thought we were going to rely on the default system encoding for these?  But, yes, I'm fine with `latin-1` if we shouldn't be using the system encoding.  These are kind of weird files because they aren't really output from stdout.. more data files created by the kernel.  I don't know if there's any documentation for how they are encoded or not.